lms/djangoapps/homepage/tests/test_custom_context_processors.py

```diff
@@ -0,0 +1,12 @@
+import mock
+
+from lms.djangoapps.homepage.constants import CONFIGS
+from lms.djangoapps.homepage.custom_context_processors import notifications_configs
+
+
+@mock.patch('lms.djangoapps.homepage.custom_context_processors.get_notifications_widget_context')
+def test_notifications_configs(mock_get_notifications_widget_context):
+    mock_get_notifications_widget_context.return_value = 'context'
+    context = notifications_configs('request')
+    mock_get_notifications_widget_context.assert_called_once_with(CONFIGS)
+    assert context == 'context'
```

Comment:
Let's please make the names more meaningful, suggestions:
1. We can rename the var on Line 10 to be `returned_context`. 
2. We can put the string value 'context' in a variable, maybe called `expected_context` and use that variable on both Line 9 and Line 12.

Reply:
I think `actual_context` would be better than `returned_context`. Nice suggestion btw.